src/main/java/test/BaseTest.java

```diff
@@ -0,0 +1,40 @@
+package test;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.util.ResourceBundle;
+import java.util.concurrent.TimeUnit;
+
+public class BaseTest {
+
+    public WebDriver driver;
+    public WebDriverWait webDriverWait;
+    private final ResourceBundle waiterTime = ResourceBundle.getBundle("resources.resoursebundle.waitertime");
```

Comment:
function System.getProperties has a realisation for default vaue in case that first value is empty, 
lets make default value read from properties and value to read-check first to be  System.getProperties, 
so you can path parameter from comand line during maven comand execution, like -Dwaitertime="some custome time"